### ISSUE TITLE
Tablets and other intel items duplicated in ACE Arsenal - magazines

### ISSUE THREAD

**Issue by SomethingSimpl** [Sparker95/Vindicta#620]
The intel items such as tablets and military documents etc. are flooding the magazines filter in ACE Arsenal with classnames that just keep adding up in the numbers. Is this by design or a bug?

![image](https://user-images.githubusercontent.com/19417313/82881759-65087d80-9f40-11ea-90a5-21789ba7d880.png)

**Comment by Sparker95** [comment]
Thanks for making the issue, I need to find out how to hide them.

Technical note: that's how it works behind the interface - each unique intel item is in fact a unique class name, ACRE and TFAR use same way to make unique radios for instance.

**Comment by SomethingSimpl** [comment]
Ah thought so! Thanks for following up anyway! maybe a `scope =` change in item config to hide from arsenal would be the best way.

**Comment by Sparker95** [comment]
Not really, intent is to open it as soon as you've got it, then it is discarded. Sometimes it can be brought to base and stored in arsenal, after loading the game it can provide invalid data, this is not handled yet.

Thanks, perhaps you are right about scopeArsenal, need to try it.

**Comment by Sparker95** [comment]
In fact if I hide them in arsenal like this, then the second problem can be solved too.

**Comment by SomethingSimpl** [comment]
Yeah, I would maybe actually go as far as to say if its plainly a gameplay related item that gets dynamically spawned and used maybe hiding it in all the scopes would be an option:
`scope = 1;` thats editor, but 0 here is used for inheritance so maybe 1 would be good.
`scopeArsenal = 0;` and `scopeCurator = 0;`.

**Comment by Sparker95** [comment]
In fact it is a bit more complicated.
We want to hide these items only in ACE and standard arsenal, but not in our arsenal. And our arsenal partially uses standard arsenal code.
And our arsenal code is something I would rather not touch unless it makes people's computers burn. Perhaps @Jeroen-Notenbomer can help.
